src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_PART_ARRAY = 0;
+    private static final int COUNT_PART_ARRAY = 1;
+    private static final String SUPPLY_CONSTANT = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        File initialFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(initialFile))) {
+            String bufferValue = bufferedReader.readLine();
+            String name = "";
+            String count = "";
+            String resultText = "";
+
+            while (bufferValue != null) {
+                name = bufferValue.split(",")[NAME_PART_ARRAY];
+                count = bufferValue.split(",")[COUNT_PART_ARRAY];
+
```

Comment:
```suggestion
            String resultText = "";

            while (bufferValue != null) {
              String name = bufferValue.split(",")[NAME_PART_ARRAY];
              String count = bufferValue.split(",")[COUNT_PART_ARRAY];

```
there is no need to declare variables outside the loop